security/keycloak-oidc-client-standalone/src/main/java/io/quarkus/ts/security/keycloak/oidcclient/standalone/TokenResource.java

```diff
@@ -0,0 +1,52 @@
+package io.quarkus.ts.security.keycloak.oidcclient.standalone;
+
+import java.util.Base64;
+
+import jakarta.inject.Inject;
+import jakarta.ws.rs.GET;
+import jakarta.ws.rs.Path;
+
+import io.quarkus.oidc.client.OidcClient;
+import io.quarkus.oidc.client.OidcClients;
+import io.quarkus.oidc.client.Tokens;
+
+@Path("/oidc-client-tokens")
+public class TokenResource {
+
+    @Inject
+    OidcClient defaultOidcClient;
+
+    @Inject
+    OidcClients oidcClients;
+
+    @Inject
+    Tokens tokens;
+
+    private Base64.Decoder decoder = Base64.getUrlDecoder();
```

Comment:
this is ok, but let's make yourself a habit to make things like these final

Reply:
Thanks I changed it.